.build-script

```diff
@@ -4,6 +4,9 @@
 # Altis Build Script #
 #====================#
 
+# Ensure build script fails if any steps return a non-zero exit code.
+set -e
```

Comment:
Let's build this into the shebang line instead. Can do `#!/bin/bash -e`

Reply:
Done